ACKategories/Classes/NSAttributedStringExtensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  NSAttributedStringExtensions.swift
+//  ACKategories
+//
+//  Created by Josef Dolezal on 04/02/2018.
+//  Copyright © 2018 Josef Dolezal. All rights reserved.
+//
+
+import Foundation
+
+precedencegroup ConcatenationGroup {
+    associativity: left
+    higherThan: AssignmentPrecedence
+}
+
+infix operator ++: ConcatenationGroup
+
+public func ++(_ lhs: NSAttributedString, _ rhs: NSAttributedString) -> NSAttributedString {
```

Comment:
I am just wondering why not use ordinal `+` operator?

Reply:
I knew someone will ask! 😄 To be honest, no special reason. It's just more transparent that it's not part of the STD/Foundation to me.

Reply:
That's a question I'd personally prefer standard `+` as all the stuff like precedence is already solved and the API is consistent.

What @pteasima thinks?

Reply:
I agree, let's use just `+` operator.